runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -0,0 +1,219 @@
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.raids;
+
+import com.google.common.eventbus.Subscribe;
+import com.google.inject.Provides;
+import java.awt.Color;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.io.InputStream;
+import java.text.DecimalFormat;
+import java.time.Instant;
+import javax.imageio.ImageIO;
+import javax.inject.Inject;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.ChatMessageType;
+import net.runelite.api.Client;
+import net.runelite.api.Varbits;
+import net.runelite.api.events.ChatMessage;
+import net.runelite.api.events.ConfigChanged;
+import net.runelite.api.events.VarbitChanged;
+import net.runelite.api.widgets.Widget;
+import net.runelite.api.widgets.WidgetInfo;
+import net.runelite.client.chat.ChatColor;
+import net.runelite.client.chat.ChatColorType;
+import net.runelite.client.chat.ChatMessageBuilder;
+import net.runelite.client.chat.ChatMessageManager;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.ui.overlay.infobox.InfoBoxManager;
+
+@PluginDescriptor(
+	name = "Raids Plugin"
+)
+@Slf4j
+public class RaidsPlugin extends Plugin
+{
+	private static final String RAID_START_MESSAGE = "The raid has begun!";
+	private static final String RAID_COMPLETE_MESSAGE = "Congratulations - your raid is complete!";
+	private static final int TOTAL_POINTS = 0, PERSONAL_POINTS = 1, TEXT_CHILD = 4;
+	private static final DecimalFormat DECIMAL_FORMAT = new DecimalFormat("###.##");
+	private BufferedImage raidsIcon;
+	private RaidsTimer timer;
+	private int raidVarbit;
+
+	@Inject
+	private ChatMessageManager chatMessageManager;
+
+	@Inject
+	private InfoBoxManager infoBoxManager;
+
+	@Inject
+	private Client client;
+
+	@Inject
+	private RaidsConfig config;
+
+	@Provides
+	RaidsConfig provideConfig(ConfigManager configManager)
+	{
+		return configManager.getConfig(RaidsConfig.class);
+	}
+
+	@Override
+	protected void startUp() throws Exception
+	{
+		updateInfoBoxState();
+	}
+
+	@Override
+	protected void shutDown() throws Exception
+	{
+		if (timer != null)
+			infoBoxManager.removeInfoBox(timer);
+	}
+
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
+	{
+		if (config.pointsMessage())
+		{
+			cacheColors();
```

Comment:
Add chatMessageManager.refreshAll() here

Reply:
the cacheColors method has .refreshAll() at the end.

Reply:
Oh sorry, totally missed that because of method chaining :man_facepalming: